### PR TITLE
[public-api] Define ProjectsService

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

Defines initial version of the ProjectsService with
* Create
* Get
* List
* Delete

## Related Issue(s)

* Closes https://github.com/gitpod-io/gitpod/issues/14606

## How to test

CI builds

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

File: components/public-api/gitpod/experimental/v1/pagination.proto [hunk 1]
```diff
@@ -5,9 +5,9 @@ package gitpod.experimental.v1;
...
 
 message Pagination {
-    // page_size is the maximum number of results we expect
+    // page_size is the maximum number of results to retrieve per page
     int32 page_size = 1;
 
-    // page_token points to a specific page of the results
-    string page_token = 2;
+    // page is the page number of results to retrieve.
+    string page = 2;
```
Comment (andrew-farries) [inline components/public-api/gitpod/experimental/v1/pagination.proto:12]: Why is this change necessary? Does it affect other services?

**Review by andrew-farries — APPROVED** [review]
One question

File: components/public-api/gitpod/experimental/v1/pagination.proto (hunk 1 above)
Comment (easyCZ) [reply components/public-api/gitpod/experimental/v1/pagination.proto:12]: Unused by other services at this point. Originally we designed as per Google API design recommendations but our existing APIs don't deal with "next_tokens" so changed it to what we do deal with today.
